src/Button.test.js

```diff
@@ -0,0 +1,34 @@
+import Button, { StyledButton } from "./Button";
+import { configure, shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import React from "react";
+
+configure({ adapter: new Adapter() });
+
+describe("Button", () => {
+  it("renders a blank button", () => {
+    expect.assertions(1);
+
+    const result = shallow(<Button />);
+
+    expect(result.debug()).toMatchSnapshot();
+  });
+  it("renders a button with text", () => {
+    expect.assertions(1);
+
+    const result = shallow(<Button children="Test" />);
+
+    expect(result.debug()).toMatchSnapshot();
+  });
+  it("executes passed function when clicked", () => {
+    expect.assertions(1);
```

Comment:
I think you only need assertions for asynchronous code

Reply:
So should we remove it from all the non-asynchronous tests?

Reply:
I think so but check with shannon as I don't use the expect.assertions stuff - So I might be wrong

Reply:
I include them in every test purely for consistency's sake but entirely up to you